toolkit/docs/building/building.md

```diff
@@ -71,58 +77,119 @@ Quick Start
+
+#### Full toolchain Bootstrap From Sources
+If you want to build **everything** from scratch, including the bootstrapping process, run:
 ```bash
 cd ~/git/CBL-Mariner/toolkit
+# Do a FULL bootstrap + rebuild from sources instead (much slower)
+# Add REBUILD_TOOLCHAIN=y to any subsequent command to ensure locally built toolchain packages are used
 sudo make toolchain REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y
 ```
+This will download the source files (SRPMs) from the package sever, and build them locally.
+See the detailed section on building from scratch [here](#building-everything-from-scratch)
 
-NOTE: A full list of targets and options to sudo make is available [here](#all-build-targets) and [here](#all-build-variables).
 
-## Build Packages
-Once the toolchain is built, all packages can be built with the toolchain.  Large parts of the build are parallelized. Enable this by setting the `-j` flag for `make` to the number of parallel jobs to allow. (Recommend setting this value to the number of hyper-threads available on your system, or less)
+### 2) Build All Packages
+(**this step may be omitted if desired**)
 
-```bash
-# Still from the toolkit folder, copy the toolchain archive built in the previous step into the toolkit folder
-mv ../build/toolchain/toolchain_built_rpms_all.tar.gz .
+Once the toolchain is bootstrapped, packages can be built with the toolchain.
+
+The image build commands in [3) Build Images](#3-build-images) will **automatically** build only the required packages based on the selected configuration.
 
+However, to manually build **all** packages you can clear the configuration with `CONFIG_FILE=` and invoke the package build target.
+
+Large parts of the build are parallelized. Enable this by setting the `-j` flag for `make` to the number of parallel jobs to allow. (Recommend setting this value to the number of logical cores available on your system, or less)
+
+**NOTE: If you are building your toolchain packages from source, add `REBUILD_TOOLCHAIN=y`**
+```bash
 # Build ALL packages FOR AMD64
-sudo make build-packages -j$(nproc) CONFIG_FILE= TOOLCHAIN_ARCHIVE=toolchain_built_rpms_all.tar.gz DOWNLOAD_SRPMS=y REBUILD_TOOLS=y REBUILD_TOOLCHAIN=n REBUILD_PACKAGES=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64"
+sudo make build-packages -j$(nproc) CONFIG_FILE= DOWNLOAD_SRPMS=y REBUILD_TOOLS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64"
 
 # Build ALL packages FOR ARM64
-sudo make build-packages -j$(nproc) CONFIG_FILE= TOOLCHAIN_ARCHIVE=toolchain_built_rpms_all.tar.gz DOWNLOAD_SRPMS=y REBUILD_TOOLS=y REBUILD_TOOLCHAIN=n REBUILD_PACKAGES=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_amd64 shim-unsigned-amd64"
+# (NOTE: CBL-Mariner compiles natively, an ARM64 build machine is required to create ARM64 packages/images)
+sudo make build-packages -j$(nproc) CONFIG_FILE= DOWNLOAD_SRPMS=y REBUILD_TOOLS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_amd64 shim-unsigned-amd64"
```

Comment:
>openjdk8_amd64 shim-unsigned-amd64 [](start = 112, length = 34)

These packages aren't needed in the ignore list. Although not necessarily harmful if present, they can be removed for clarity.

Reply:
Removed